test_conformance/vulkan/test_vulkan_interop_buffer.cpp

```diff
@@ -640,16 +612,14 @@ int run_test_with_one_queue(cl_context &context, cl_command_queue &cmd_queue1,
                                 " error\n");
                     goto CLEANUP;
                 }
-                if (iter != (maxIter - 1))
+                if (use_fence)
```

Comment:
Can you propagate fence improvements everywhere in the file where fences were added?

Reply:
Done